vocabularies/security-classifications.ttl

```diff
@@ -0,0 +1,71 @@
+ skos:definition "OFFICIAL represents most Queensland Government information by volume, but lowest business impact per document if compromised or lost. However, where information is aggregated on an information asset such as an ICT server, the impact of compromise may increase and with it, the controls.  OFFICIAL information is routine information without special sensitivity or handling requirements. All routine public-sector business, operations and services is treated as OFFICIAL. At the OFFICIAL classification there is a general presumption that data may be shared across government. Security measures should be proportionate and driven by the business requirement.  Most OFFICIAL information is subject to the Public Records Act 2002 (Qld)."@en ;
+ skos:inScheme <http://linked.data.gov.au/def/qg-security-classifications> ;
+ skos:notation "off" ;
+ rdfs:label "OFFICIAL"@en ;
+ skos:prefLabel "OFFICIAL"@en  .
+
+qgiscf:official-public a skos:Concept ; 
+ skos:broader qgiscf:official ;
+ skos:definition "As per OFFICIAL with the Dissemination Limiting Marking (DLM) applied to indicates the information is publically available and may be accessed by or released to the public."@en ;
+ skos:inScheme <http://linked.data.gov.au/def/qg-security-classifications> ;
+ skos:notation "ofp" ;
+ rdfs:label "OFFICIAL-PUBLIC"@en ;
+ skos:prefLabel "OFFICIAL-PUBLIC"@en  .
+
+qgiscf:sensitive a skos:Concept ;  
+ skos:topConceptOf <http://linked.data.gov.au/def/qg-security-classifications> ; 
+ skos:definition "The use of the SENSITIVE indicates that information requires additional handling care due to its sensitivity or moderate business impact if compromised or lost.  SENSITIVE information must be labelled.  Examples of SENSITIVE information may include: government or agency business, whose compromise could affect the government's capacity to make decisions or operate, the public's confidence in government, the stability of the market place and so on commercial interests, whose compromise could significantly affect the competitive process and provide the opportunity for unfair advantage legal professional privilege law enforcement operations whose compromise could adversely affect crime prevention strategies, particular investigations or adversely affect personal safety personal information, which is required to be safeguarded under the Information Privacy Act 2009 (Qld), or other legislation."@en ;
+ skos:inScheme <http://linked.data.gov.au/def/qg-security-classifications> ;
+ skos:notation "sns" ;
+ rdfs:label "SENSITIVE"@en ;
+ skos:prefLabel "SENSITIVE"@en  .
+
+qgiscf:protected a skos:Concept ;
+ skos:topConceptOf <http://linked.data.gov.au/def/qg-security-classifications> ;
+ skos:definition "A pre-existing related resource that is substantially the same as the described resource, but in another format."@en ;
+ skos:inScheme <http://linked.data.gov.au/def/qg-security-classifications> ;
+ skos:notation "prtc" ;
+ rdfs:label "PROTECTED"@en ;
```

Comment:
All caps?

Reply:
It's how it's listed in the policy guideline (for DES at least)